code/modules/client/preferences.dm

```diff
@@ -105,9 +105,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		load_path(parent.ckey)
 		if(load_and_save && !fexists(path))
 			try_savefile_type_migration()
-		unlock_content = !!parent.IsByondMember()
-		if(unlock_content)
-			max_save_slots = 8
+		unlock_content = TRUE
```

Comment:
bu 8 kalabilir bence

Reply:
ileride patreon perki icin dusundum onu + disk konusunda problemlerimiz var zaten

Reply:
patreon için olabilir mantıklı. disk için ayda giren oyuncu sayısıyla (361) hesapladığımda ~5MB farkediyor her birinin 5 yeni karakter oluşturması